code/obj.dm

```diff
@@ -332,7 +332,7 @@
 
 
 /obj/lattice
-	desc = "A lightweight support lattice."
+	desc = "A series of intersecing metal rods, used as a structural skeleton for space stations and to facilitate movement in a vacuum."
```

Comment:
```suggestion
	desc = "A lightweight support lattice which can be used to facilitate movement in a vacuum."
```
this description felt needlessly complicated to me for what it is, kept important info